### PR TITLE
[ADD] website_event_register

### PR DESCRIPTION
**Pull request by hbrunn**
# Register for free website events

This module allows users to register for free events. Without this module, you'd have to install website_event_sale that has a lot of possibly unnecessary dependencies, define tickets for 0 euros and still 'sell' them to users, resulting in sale orders with zero amount and generally a misleading user experience.
# Usage

After installation, a registration link will show up for published events with free seats. If you happen to have website_event_sale installed, this link will only show if there are no tickets defined for the event.
# Known issues / Roadmap
- it would be nice if we could define tickets with price zero and free registration then asks which of the free tickets we want to take.

### CODE REVIEW

File: website_event_register_free/model/ir_ui_view.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+from openerp import models, api
+
+
+class IrUiView(models.Model):
+    _inherit = 'ir.ui.view'
+
+    @api.cr_uid_ids_context
+    def render(self, cr, uid, id_or_xml_id, values=None, engine='ir.qweb',
+               context=None):
+        '''inject functions we need in out template'''
```
Comment (pedrobaeza) [inline website_event_register_free/model/ir_ui_view.py:30]: Use double quotas for documenting.
s/out/our/?

**Comment by pedrobaeza** [comment]
A fast code review says me that :+1:

**Comment by guewen** [comment]
:+1: 
Corrected the typo and double quote things during merge

**Comment by guewen** [comment]
Oops:

```
$ git push                                                                                                                                                                  8.0 
ERROR: Permission to OCA/website.git denied to guewen.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

I miss the rights on this repo

**Comment by hbrunn** [comment]
@guewen now I think you should be able to merge this

**Comment by guewen** [comment]
Done!

**Comment by hbrunn** [comment]
thanks!

**Pull request merged**